modules/default-requirements-external-db.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-default-requirements.adoc
+:_mod-docs-content-type: CONCEPT
+[id="external-db-req_{context}"]
+= Requirements for using an external database
+
+[role="_abstract"]
+You can configure {product-title} Central services to use an external PostgreSQL-compatible database for data persistence instead of deploying its own database pod.
+
+[IMPORTANT]
+====
+When you use an external database:
+
+* Its infrastructure manages persistent storage for Central DB. Therefore, do not configure persistence settings for the Central DB within the {product-title-short} installation configuration.
+* Red Hat supports the configuration and operation of {product-title-short} Central connected to that database. However, support for database-specific operations such as backup and restore, performance diagnosis and potential tuning, software and version upgrades, and HA/DR operation falls under 3rd Party Support. Manually upgrading or customizing your database outside of a full platform upgrade also limit supportability.
+====
+
+If you select an external database, your database instance and the user connecting to it must meet the following requirements:
```

Comment:
Since these are headings that follow, I think that this should be a complete sentence. Maybe:

```suggestion
If you select an external database, your database instance and the user connecting to it must meet the requirements listed in the following sections.
```